src/WaterPokemon.java

```diff
@@ -0,0 +1,21 @@
+public class WaterPokemon extends Pokemon {
+
+    private String badkleding;
+
+    public WaterPokemon(String swimPokemon, String breathPokemon, String expertise, String badkleding)
```

Comment:
Je keuze voor parameter-namen is hier niet logisch. Wat je hier "swimPokemon" noemt is in de constructor van Pokemon de "soundPokemon". Je kunt het hier dan ook makkelijk "soundPokemon" noemen